Makefile

```diff
@@ -32,6 +32,9 @@ clean:
 mod-tidy:
 	export GO111MODULE=on; go mod tidy
 
+release:
```

Comment:
Maybe `helm-release` or `helm-publish` or something like that? Just don't want this to be confused with actually cutting a new release.

Reply:
Added some docs around creating releases to make sure I (and others) don't miss anything in the future. I want this task to auto-create the release at some point...

Actually maybe I'll script out the necessary git commands.

Reply:
Gotcha that totally makes sense. Releases would probably be pretty easy to script out with the github cli, but that's a different PR.